### ISSUE TITLE
Provide a destroy method.

### ISSUE THREAD

**Issue by GeorgMeyer23** [interactive-pioneers/iptools-jquery-modal#3]
Provide a destroy method that at least removes plugin data and event listeners.

example: (see: https://github.com/interactive-pioneers/iptools-jquery-expandable_module)
```javascript
destroy: function() {
  this.$element.off('click' + this.settings.eventNamespace, '.' + cssClassTrigger);
  this.$element.removeData();
}
```

**Comment by GeorgMeyer23** [comment]
Also remove the created div 'this.$modal'.

**Comment by ain** [comment]
@davleh you can automagically close by commit message ;) 

E.g.:
- Resolve #3 Provide a destroy method
- Fix #3 Destroy method missing
